### ISSUE TITLE
Standard surveillance output

### ISSUE THREAD

**Issue by seabbs** [CDCgov/Rt-without-renewal#165]
I think we want to define some standard surveillance output for `_make_epi_aware` as this will help with postprocessing and be of general utility (given the aim of the tooling). 

I think the way to we want to do this is to create a new abstract function in `EpiAwareBase` that is called in the return of `_make_epi_aware` and dispatches on the `EpiInfModel` type used. This is so that we can have a generic method that just uses the expected infections for its metrics and more specific methods that use the already computed reproduction number estimates and growth rate calculations. 

Alternatively, we could consider this as a post-processing step for the model output (i.e it works with a for object) but I think its nice to make use of the struct information we have to provide custom solutions which I think favours doing this in model (albeit only as part of the generated quantities).

**Comment by SamuelBrand1** [comment]
I think this is a really good idea. Should be reasonable to achieve, we want or already have the functions to fill this in.

We can also add a pretty print to the struct so you get a lot of information (Unicode plots even?)

**Comment by seabbs** [comment]
> We can also add a pretty print to the struct so you get a lot of information 

I think we want more pretty printing of structs in general and wondered if there was a package out there. Can't find anything right now though

**Comment by seabbs** [comment]
Aside from structs are there any other data structures (and links please) we could be thinking about here to use to control dispatch?

**Comment by SamuelBrand1** [comment]
> > We can also add a pretty print to the struct so you get a lot of information
> 
> I think we want more pretty printing of structs in general and wondered if there was a package out there. Can't find anything right now though

I think the canonical way of doing this is overloading [`Base.show`](https://docs.julialang.org/en/v1/base/io-network/#Base.show-Tuple{IO,%20Any,%20Any}) with a custom method

**Comment by seabbs** [comment]
Yeah I saw this and I was wondering if anyone had already done work on that for this kind of use case

**Comment by seabbs** [comment]
This kind of thing: https://github.com/MechanicalRabbit/PrettyPrinting.jl
